include/inexor/vulkan-renderer/availability_checks.hpp

```diff
@@ -9,77 +9,57 @@
 namespace inexor::vulkan_renderer {
 
 class AvailabilityChecksManager {
-private:
```

Comment:
Removed private?

Reply:
`class` is always private by default.

Reply:
But you are right, I will remove this change since it's not in the scope of this pull request.

Reply:
Done.